R/recipe.R

```diff
@@ -414,20 +424,36 @@ prep.recipe <-
 
     running_info <- x$term_info %>% mutate(number = 0, skip = FALSE)
 
+    needs_tuning <- purrr::map(x$steps, ~ {
+      res <- map_lgl(.x, is_tune)
+      res <- names(res)[res]
+      tibble(step = class(.x)[[1L]], arg = res)
+    })
+    needs_tuning <- purrr::list_rbind(needs_tuning)
+
+    if (nrow(needs_tuning) > 0) {
+      args <- vctrs::vec_split(needs_tuning$arg, needs_tuning$step)
+      msg <- c(
+        x = "You cannot {.fun prep} a tuneable recipe.",
```

Comment:
```suggestion
        x = "You cannot {.fun prep} a tunable recipe.",
```

I believe we spell it like this consistently elsewhere. If many exceptions, feel free to ignore.